db/setup/db-creation/db-creation.py

```diff
@@ -0,0 +1,83 @@
+import psycopg
+import os
+
+NACHET_DB_URL = os.getenv("NACHET_DB_URL")
```

Comment:
I'm unsure of the value of scripting creation with Python here instead of having a .sql file and using psql to do the loading? Especially as I don't really see any error handling.

I wrote some loading scripts for that purpose:

https://github.com/ai-cfia/ailab-db/blob/e2fefdf3b4142d46f2aa6dfe2adaf2ef807facfc/sql/schema2.sql.public_new.sql#L2

Reply:
This file was mainly for my own usage for when I was in the office and had to update the schema. I expect to change our way of working if we migrate with Bytebase. I'm going to set up a pipeline soon